packages/conventional-changelog-cli/README.md

```diff
@@ -36,7 +36,7 @@ To fully customize the tool, please checkout [conventional-changelog](https://gi
 
 ## Example output
 
-- https://github.com/ajoslin/conventional-changelog/blob/master/CHANGELOG.md
+- https://github.com/conventional-changelog/conventional-changelog/blob/master/packages/conventional-changelog/CHANGELOG.md
```

Comment:
@eddiejaoude it looks like you originally were pointing to the `cli` package, but switched to the normal `conventional-changelog` package. Any particular reason? (I would assume the `cli` package would point to its own `CHANGELOG.md` file as an example)

Reply:
@hbetts the original repo link `ajoslin/conventional-changelog` redirects to `conventional-changelog/conventional-changelog`, so I used the changelog from `conventional-changelog` package.

Happy to change if that is what is preferred, but the `404` was definitely not useful 🤓

Reply:
> the original repo link ajoslin/conventional-changelog

Oh, no, that's actually a good point. Let's stick with the original intent.

We can change the URL to point to the CLI at a later time.